vivarium/core/emitter.py

```diff
@@ -251,9 +257,13 @@ def emit(self, data: Dict[str, Any]) -> None:
         if data['table'] == 'history':
             emit_data = data['data']
             time = emit_data['time']
-            self.saved_data[time] = {
+            time_data = {
```

Comment:
this can easily be interpreted as "data about time". Can we call it `data_at_time`?